### PR TITLE
364 about page

### LINKED ISSUE #364: [FRONTEND] Create about page
**Is your feature request related to a problem? Please describe.**
Use the components made in #351 

Image

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #364

### CODE REVIEW

**Comment by zlrkw11** [comment]
image
image
- small viewport

**Comment by zlrkw11** [comment]
I replaced the old images with new ones I found on UASC facebook because the resolution was too low for testing purposes. It will be responsive so we should change the images according to figma design once those photos are provided (not screenshots).

**Comment by zlrkw11** [comment]
I have also decided to cancel the overlapping design on the textbox. It makes the webpage glitch a lot and making it hard to style.
Now the textbox will stay under the image for both md and lg viewports.

image
- md viewport

image
- lg viewport

**Comment by bcho892** [comment]
> I have also decided to cancel the overlapping design on the textbox. It makes the webpage glitch a lot and making it hard to style. Now the textbox will stay under the image for both md and lg viewports.
> 
> image
> 
> * md viewport
> 
> image
> 
> * lg viewport

That is fine to not make them overlapping, however, I'd prefer you to instead put them side by side as we are wasting a lot of space right now

File: client/src/components/generic/AboutSection/AboutSection.tsx [hunk 1]
```diff
@@ -8,12 +8,12 @@ interface IAboutProps {
 }
 
 type Props = IAboutProps
```
Comment (bcho892) [inline client/src/components/generic/AboutSection/AboutSection.tsx:10]: `IAboutProps` is a confusing name, should name it after the component

File: client/src/components/generic/AboutSection/AboutSection.tsx [hunk 2]
```diff
@@ -24,29 +24,25 @@ const TextStyler = ({ title, text }: Props) => {
...
-            variant={"left"}
-            imageSrc={imageSrc}
-          />
+      <div className="w-full">
+        <img
+          src={imageSrc}
+          className="relative w-full rounded-t-lg md:h-[500px] md:w-[900px]"
+        ></img>
+        <div className=" border-dark-blue-100 flex w-full flex-col rounded-b-lg border bg-white md:ml-auto md:w-[50%] md:rounded-lg lg:ml-[40%] lg:w-[530px] lg:rounded-t-lg">
+          <TextStyler title={title} text={text} />
```
Comment (bcho892) [inline client/src/components/generic/AboutSection/AboutSection.tsx:33]: What text? Name could be more descriptive

**Review by bcho892 — CHANGES_REQUESTED** [review]

File: client/src/components/generic/AboutSection/AboutSection.tsx [hunk 3]
```diff
@@ -24,29 +24,25 @@ const TextStyler = ({ title, text }: Props) => {
...
-          <TextStyler
-            title={title}
-            text={text}
-            variant={"left"}
-            imageSrc={imageSrc}
-          />
+      <div className="w-full">
+        <img
+          src={imageSrc}
+          className="relative w-full rounded-t-lg md:h-[500px] md:w-[900px]"
```
Comment (bcho892) [inline client/src/components/generic/AboutSection/AboutSection.tsx:30]: your images are stretching and changing aspect ratio, look into https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit `object-fit: cover`

**Comment by bcho892** [comment]
Can we set a max width for the page (900-1000px is probably good), also prefer text to be vertically centered:

image

**Comment by bcho892** [comment]
this image gets stretched on larger screens:

image

**Comment by zlrkw11** [comment]
image
image
image
- responsive for 3 viewports
- right section conditionally renders
- space used increased
- styling improved
- left and right sections adjusted to stay same in styling

**Review by bcho892 — APPROVED** [review]

**Pull request merged**